src/07-uart/send-a-single-byte.md

```diff
@@ -0,0 +1,48 @@
+# Send a single byte
+
+Our first task will be to send a single byte from the microcontroller to the computer over the serial
+connection.
+
+In order to do that we will use the following snippet (this one is already in `07-uart/src/main.rs`):
+
+``` rust
+{{#include src/main.rs}}
+```
+
+
+There is some parts we have already seen before but also lots of new stuff so lets focus on those.
+First things first, how do we know that we have to interact with P0_24 and P0_25 here? If you take a look
+at the micro:bit schematics page 5 again, you will notice that those two pins are also referred to as TGT_RX
+and TGT_TX. Not only that, they are also connected to our debug probe on page 2 so those are most likely the two
+pins we are looking for.
+
+Next up we can observe a new pin configuration we haven't seen before `p0.p0_25.into_floating_input()`, this just
+means that P0_25 is now an input, what exactly floating means is none of our concern right now. But how do we know
+we have to put our pins in this exact mode? If you look at the line below you can see that we construct an instance
+of `Pins` which is later passed on to our UART peripheral constructor (cts and rts are for more advanced UART features
```

Comment:
```suggestion
of `Pins` which is later passed on to our UART peripheral constructor (`cts` and `rts` are for more advanced UART features
```
I'm guessing. Otherwise "cts" and "rts" in quotes I would say.